Tools/cases_generator/flags.py

```diff
@@ -53,10 +55,11 @@ def names(self, value: bool | None = None) -> list[str]:
             return list(dataclasses.asdict(self).keys())
         return [n for n, v in dataclasses.asdict(self).items() if v == value]
 
-    def bitmap(self) -> int:
+    def bitmap(self, ignore: list[str] = []) -> int:
```

Comment:
Using a mutable default sets alarm bells off in my head, even though you're safe in this case. Maybe make it an abstract set?